packages/forms/test/reactive_integration_spec.ts

```diff
@@ -2304,6 +2304,93 @@ const ValueAccessorB = createControlValueAccessor('[cva-b]');
            expect(resultArr.length)
                .toEqual(2, `Expected original observable to be canceled on the next value change.`);
          }));
+
+
+      it('should validate max', () => {
+        const fixture = initTest(FormGroupMinMaxComp);
+        fixture.componentInstance.form = new FormGroup({'pin': new FormControl(5)});
+        fixture.detectChanges();
+
+        const input = fixture.debugElement.query(By.css('input'));
+        expect(input.nativeElement.value).toEqual('5');
```

Comment:
We can retrieve `nativeElement` sooner and just just `input` below (in a few places):
```suggestion
        const input = fixture.debugElement.query(By.css('input')).nativeElement;
        expect(input.value).toEqual('5');
```